irods/test/helpers.py

```diff
@@ -96,14 +100,16 @@ def make_session(**kwargs):
             env_file = os.environ['IRODS_ENVIRONMENT_FILE']
         except KeyError:
             env_file = os.path.expanduser('~/.irods/irods_environment.json')
+    session = iRODSSession( irods_env_file = env_file, **kwargs )
 
-    try:
-        os.environ['IRODS_CI_TEST_RUN']
-        uid = getpwnam('irods').pw_uid
-    except KeyError:
-        uid = None
+    connected_vsn = session.server_version[:3]
+    advertised_vsn = IRODS_VERSION[:3]
+    if connected_vsn > advertised_vsn:
+        msg = ("Connected server is {connected_vsn}, "
+               "but this python-irodsclient is compatible up to {advertised_vsn}.").format(**locals())
```

Comment:
use all the letters?

"connected_vsn" -> "connected_version"
and
"advertised_vsn" -> "advertised_version"